doc/src/modules/physics/mechanics/examples/four_bar_linkage_example.rst

```diff
@@ -0,0 +1,61 @@
+==================
+A four bar linkage
+==================
+
+The four bar linkage is a common example used in mechanics, which has two holonomic constraints. This example will make use of joints functionality provided in :mod:`sympy.physics.mechanics`. In summary we will use bodies and joints to define the open loop system. Next, we define the configuration constraints to close the loop. The ``JointsMethod`` will be used to do the "book-keeping" of the open-loop system. From this we will get the input used in combination with the constraints to manually setup the :class:`~.KanesMethod` as the backend.
```

Comment:
".., which can be formulated with only two explicit holonomic constraints" is a bit more correct.